source/content/guides/drupal-to-build-tools.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: Migrate a Composer-based Drupal Site to a Build Tools Site
-description: Migrate a Drupal 9 site created via Pantheon Dashboard (or Terminus) to a Build Tools-based site.
+description: Migrate a Drupal 8.8+ site created via Pantheon Dashboard (or Terminus) to a Build Tools-based site.
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-to-build-tools